### PR TITLE
ID75 v2: add default keymap; update info.json

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

This keyboard was submitted without a default keymap. :confused:

Also updates some of the `info.json` data.

cc @peepeetee (keyboard maintainer)

## Types of Changes

- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/idobao/id75/v2/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 QMK / James Young (@noroadsleft)
```
Comment (zvecr) [inline keyboards/idobao/id75/v2/keymaps/default/keymap.c:1]: Does `keyboards/idobao/id75/keymaps/default/keymap.c` not cover this case?

Comment (noroadsleft) [reply keyboards/idobao/id75/v2/keymaps/default/keymap.c:1]: Hmm, I need to look into my local tooling then. I have some local scripts which check for this, but this was the only keyboard it didn't find a default keymap for.

Comment (peepeetee) [reply keyboards/idobao/id75/v2/keymaps/default/keymap.c:1]: The keyboard was submitted without a default keymap for this reason, as I reasoned that the top level one would suffice

**Comment by peepeetee** [comment]
The info.json update looks great, should we open a PR for that?

File: keyboards/idobao/id75/v2/keymaps/default/keymap.c (hunk 1 above)
Comment (noroadsleft) [reply keyboards/idobao/id75/v2/keymaps/default/keymap.c:1]: The top level one does work, though looking at the docs Idobao provides for the v2, the v2 keymap isn't the same as we currently have for the v1.

**Pull request closed without merging**